### DISCUSSION TITLE
Project Hierarchy and Discussion

### DISCUSSION THREAD

**Discussion by AIIX** [OpenVoiceOS/ovos-buildroot#111] [Ideas]
This is a discussion to categorise and put all repositories living in the OpenVoiceOS organisation into a hierarchy levels based on their usage:

There can be generally 3 levels of categorisation for example:
Level 1: Main and critical projects that are to some extent self independent
Level 2: Direct project dependencies of the critical projects required for full functionality of the main project
Level 3: Addons and components that supplement level 2 projects
 
Level one projects are denoted by a single - |  Level two projects are denoted by double -- | Level three projects are denoted by triple dash ---

-----------------------------------------------------------------------

- [Ovos-Core](https://github.com/OpenVoiceOS/ovos-core) 
  -- [Ovos-Phal](https://github.com/OpenVoiceOS/ovos_phal)
  --- [Ovos-Phal-Plugin-System](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-system)

**Comment by j1nx** [comment]
Thanks for that.

@JarbasAl had already prepared some awasome-lists. Let me try to find them and use them for input.

**awesome-ocp-skills**
https://github.com/OpenVoiceOS/awesome-ovos-plugins

**awasome-ovos-plugins**
https://github.com/OpenVoiceOS/awesome-ovos-plugins

**Comment by JarbasAl** [comment]
this gets tricky fast... 
- PHAL can be used without core eg hivemind voice-sat goes with PHAL, so it should not go under ovos-core
- OCP can also be used standalone, you can run it as a systemservice or like a audio plugin, im putting it behind a flask server for demo purposes of the search functionality, so is this top level, or right at the bottom with plugins?
